test/model.js

```diff
@@ -234,13 +232,13 @@ describe('Model', () => {
 
   describe('.saveWithId()', function() {
     it('should save with a given ID', async () => {
-      const id = new bson.ObjectID('1234abcd103f8e485c9d2019');
+      const id = new bson.ObjectID('1234abcd103f8e485c9d2017');
```

Comment:
why this change?

Reply:
I changed this when I was getting duplicate key errors I'll revert.